src/Main.purs

```diff
@@ -4,45 +4,46 @@ module Main
 
 import Prelude
 
+import Data.Array as Array
 import Data.Maybe (Maybe(..))
 import Data.Tuple.Nested ((/\))
 import Effect (Effect)
 import Effect.Exception (throw)
 import React.Basic.DOM as React.DOM
 import React.Basic.DOM.Client (createRoot, renderRoot)
-import React.Basic.Events as React.Events
 import React.Basic.Hooks (Component, component)
 import React.Basic.Hooks as React
+import Wallet (Wallet)
 import Web.DOM (Element)
 import Web.DOM.NonElementParentNode (getElementById)
-import Web.HTML (HTMLDocument, window)
+import Web.HTML (HTMLDocument, Window, window)
 import Web.HTML.HTMLDocument (toNonElementParentNode)
 import Web.HTML.Window (document)
 
-mkCounter :: Component Int
-mkCounter = do
+newtype StartPageState = StartPageState (Array Wallet)
```

Comment:
@yveshauser The user can have zero or more wallets installed which needs to be handled.